Backpack/Calendar/Classes/BPKCalendarPriceLabelCell.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKCalendarCell.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class BPKLabel;
+
+/**
+ * `BPKCalendarPriceLabelCell` is a subclass of `BPKCalendarCell`. It allows a custom label containing auxillary price information to be shown below the cell.
+ */
+@interface BPKCalendarPriceLabelCell : BPKCalendarCell
+
+/**
+ * The label displayed at the bottom of the cell.
+ */
+@property(nonatomic, strong, nonnull, readonly) BPKLabel *priceLabel;
```

Comment:
`nonnull` is redundant because of `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END`